tests/tests.sh

```diff
@@ -9,7 +9,7 @@ run_test() {
     cd "${1}files" || return 3
     output=$(../../../check-trailing-whitespaces)
     status_code="$?"
-    cd ../.. || return 3
+    cd "$OLDPWD" || return 3
```

Comment:
wow

Reply:
This is equivalent to `cd - > /dev/null`.